src/e3/vcs/git.py

```diff
@@ -92,25 +95,34 @@ def create(cls, repo_path: str, initial_content_path: Optional[str] = None) -> s
             repo.git_cmd(["commit", "-m", "initial content"])
         return repo_path
 
-    def git_cmd(self, cmd: Git_Cmd, **kwargs) -> Run:
+    def git_cmd(
+        self,
+        cmd: Git_Cmd,
+        output: Union[
+            STDOUT_VALUE, DEVNULL_VALUE, PIPE_VALUE, GIT_LOG_STREAM_VALUE, str, IO, None
```

Comment:
Can't we create an intermediate type Run_Output and in this command have Union[Run_Output, GIT_LOG_STREAM_VALUE]

Reply:
I'm not really convinced that this is a good idea, what we have here is useful I think for documenting the accepted values